app/javascript/components/server-components/CheckoutDashboard/SocialProofWidgetsPage.tsx

```
@@ -0,0 +1,1203 @@
+];
+
+type Product = {
+  id: string;
+  name: string;
+  url: string;
+  thumbnail_url?: string | null;
+  sales_count: number;
+  price: string;
+};
+
+type SocialProofWidget = SocialProofWidgetType;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'SocialProofWidget'. Consider to delete it or rename it.

'SocialProofWidget' is defined here:


(lint/suspicious/noRedeclare)